packages/apps/dashboard/server/src/modules/details/details.controller.ts

```diff
@@ -49,11 +48,11 @@ export class DetailsController {
   public async leaders(
     @Query('chainId') chainId?: ChainId,
   ): Promise<LeaderDto[]> {
-    return this.detailsService.getBestLeadersByRole(chainId);
+    return this.detailsService.getLeadersByChainId(chainId, 4);
```

Comment:
Before this PR it was 5 items in leaders table, so just in case
```suggestion
    return this.detailsService.getLeadersByChainId(chainId, 5);
```

Reply:
It is 4
![image](https://github.com/user-attachments/assets/62fbcbcb-c416-4f22-8675-b4067e171a7c)

Reply:
Sorry, I confused with "all leaders" list